src/org/mirah/builtins/builtins.mirah

```diff
@@ -20,25 +20,31 @@ import java.util.Collections
 
 class Builtins
   def self.initialize_builtins(type_system:TypeSystem)
-    type_system.extendClass('java.util.Collection', CollectionExtensions.class)
-    type_system.extendClass('java.util.Map', MapExtensions.class)
-    type_system.extendClass('java.util.List', ListExtensions.class)
-    type_system.extendClass('java.lang.Object', ObjectExtensions.class)
-    type_system.extendClass('java.lang.Iterable', EnumerableExtensions.class)
-    type_system.extendClass('java.lang.Iterable', IterableExtensions.class)
-    type_system.extendClass('java.lang.String', StringExtensions.class)
-    type_system.extendClass('java.lang.StringBuilder', StringBuilderExtensions.class)
-
-    type_system.extendClass('java.util.concurrent.locks.Lock', LockExtensions.class)
-    type_system.extendClass('java.util.regex.Matcher', MatcherExtensions.class)
-
-    type_system.extendClass('int', IntExtensions.class)
-    type_system.extendClass('byte', NumberExtensions.class)
-    type_system.extendClass('short', NumberExtensions.class)
-    type_system.extendClass('int', NumberExtensions.class)
-    type_system.extendClass('long', NumberExtensions.class)
-    type_system.extendClass('float', NumberExtensions.class)
-    type_system.extendClass('double', NumberExtensions.class)
+    if self.builtins_enabled 
+      type_system.extendClass('java.util.Collection', Class.forName("org.mirah.builtins.CollectionExtensions"))
```

Comment:
We've had Class.forName() here before (".class" is just a shorthand for Class.forName()), and that error doesn't seem to happen in practice, and never will for a user (as "mirahc-naked.jar" is only used internally in the "mirah" project). Apart from that, the stack trace would show where the problem is. So I do not think wrapping exceptions here is necessary or even useful.